src/components/mediaImage/template.tsx

```diff
@@ -83,6 +83,7 @@ export const MediaImage = (
       className={props.className}
       style={props.style}
       loading={props.loading}
+      fetchPriority="high"
```

Comment:
I don't know if this actually helps at all, but my dev tools said to make it happen, so I did. I didn't notice any difference.

It certainly doesn't help with the CLS, but I kept it anyhow. Happy to revert this if y'all think it's not a good idea.